pipelines/azure_command_center.py

```diff
@@ -210,11 +211,14 @@ def do_pyrenew_reruns(
             locations_include=locations_include,
             exclude_last_n_days=h_exclude_last_n_days,
         )
-        fit_pyrenew_hw(
-            append_id=append_id,
-            locations_include=locations_include,
-            exclude_last_n_days=h_exclude_last_n_days,
-        )
+
+        if WASTEWATER_DISEASES:
+            fit_pyrenew_hw(
+                append_id=append_id,
+                locations_include=locations_include,
+                exclude_last_n_days=h_exclude_last_n_days,
+                diseases=WASTEWATER_DISEASES,
+            )
```

Comment:
I think better to revert these and change the `fit_pyrenew_hw = partial(` definitions.